tests/cypress/integration/actions_users/registration_involved/case_28_review_pipeline_feature.js

```diff
@@ -351,25 +351,23 @@ context('Review pipeline feature', () => {
         });
 
         it('Select an issue on sidebar. Issue indication has changed the color for highlighted issue', () => {
-            let index = 0;
             cy.collectIssueRegionId().then(($issueRegionList) => {
-                cy.get('.cvat-objects-sidebar-issue-item').then((sidebarIssueItems) => {
-                    for (let i = 0; i < sidebarIssueItems.length; i++) {
-                        cy.get(sidebarIssueItems[i]).trigger('mousemove').trigger('mouseover');
-                        cy.get(`#cvat_canvas_issue_region_${$issueRegionList[index]}`).should(
-                            'have.attr',
-                            'fill',
-                            'url(#cvat_issue_region_pattern_2)',
-                        );
-                        cy.get(sidebarIssueItems[i]).trigger('mouseout');
-                        cy.get(`#cvat_canvas_issue_region_${$issueRegionList[index]}`).should(
-                            'have.attr',
-                            'fill',
-                            'url(#cvat_issue_region_pattern_1)',
-                        );
-                        index++;
-                    }
-                });
+                for (let i = 0; i < $issueRegionList.length; i++) {
+                    cy.get(`#cvat-objects-sidebar-issue-item-${$issueRegionList[i]}`)
```

Comment:
`#cvat-objects-sidebar-issue-item-${issueRegionID}`

Reply:
The same below

Reply:
Done.